hlx_statics/blocks/inlinealert/inlinealert.js

```diff
@@ -55,8 +56,10 @@ export default async function decorate(block) {
         // TODO: may need to refactor this logic
         let classVariant = getVariant(block.parentElement.parentElement.classList) ;
         if(classVariant) {
-            inlineAlert.classList.add(classVariant.class);
-            inlineAlert.insertAdjacentHTML("afterbegin", classVariant.icon);
+            const inlineClass = classVariant.class ? classVariant.class : 'spectrum-InLineAlert--info';
+            const inlineIcon = classVariant.class ? classVariant.icon : infoIcon;
```

Comment:
should this be

```suggestion
            const inlineIcon = classVariant.icon ? classVariant.icon : infoIcon;
```

Reply:
Ah @melissag-ensemble is correct here, I believe.

Reply:
@timkim, @melissag-ensemble, I have updated the changes, kindly review them.